app/websocket/app/main/routes.py

```diff
@@ -22,6 +23,51 @@
 # worker
 MODEL_LIMIT_SIZE = (1024 ** 2) * 100  # 100MB
 
+# ======= WEB ROUTES ======
+# TOOD: Must verify if cores block these calls or not. Should be protected by cors.
```

Comment:
TOOD -> TODO. @IonesioJunior can you check this?

Reply:
Ok. I'm on it.

Reply:
Currently, we enable Cross-Origin Resources to all routes. We can fix this in another Pull request.